index.html

```diff
@@ -2881,15 +2881,15 @@ <h2 id="docconformance">
           <tr id="att-contenteditable" tabindex="-1">
             <td>
               Element with <a data-cite=
-              "html/interaction.html#attr-contenteditable">`contenteditable`</a>
-              attribute
+              "html/interaction.html#attr-contenteditable">`isContentEditable`</a>
```

Comment:
agree with the update to add clarity, but would rather this be added as a description. Otherwise this would be the only instance of directly referring to an IDL attribute.

```suggestion
              "html/interaction.html#attr-contenteditable">`contenteditable`</a> where `isContentEditable="true"`
```

Reply:
The text now reads

> Element with `contenteditable` where `isContentEditable="true"`.

This sounds like only `first` should have `aria-readonly="false"` in the following example, but `contenteditable="inherit"` is the default so first and second should be identical.

```html
<div contenteditable="true">
  <div id="first" contenteditable="inherit"></div>
  <div id="second"></div>
<div>
```

I think the IDL attribute represents the correct semantics, an alternative to directly referring to it would be to describe its semantics, something like

> Element with `contenteditable="true"`; or
> Element without a `contenteditable` attribute whose closest ancestor with a `contenteditable` attribute has `contenteditable="true"`

What do you think?

Reply:
I like your new proposed text.  Let's go with that.